internal/app/syslogd/parser/parse.go

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package parser provides a syslog parser that can parse both RFC3164 and RFC5424 with best effort.
+package parser
```

Comment:
nit, but I'd move to `internal/app/syslogd/internal/parser/parse.go` so that it's private part of this service implementation